doc/async_worker_variants.md

```diff
@@ -243,7 +243,9 @@ be safely released.
 Note that `Napi::AsyncProgressWorker::ExecutionProcess::Send` merely guarantees
 **eventual** invocation of `Napi::AsyncProgressWorker::OnProgress`, which means
 multiple send might be coalesced into single invocation of `Napi::AsyncProgressWorker::OnProgress`
-with latest data.
+with latest data. If you would like to guarantee that there is one invocation of
+`Napi::AsyncProgressQueueWorker::OnProgress` for every `Send` call, you should use the
```

Comment:
Maybe it could be:
If you would like to guarantee that there is one invocation of `OnProgress` for every `Send` call, ...

Reply:
Sure, no issue shortening it.